src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -6834,16 +6834,19 @@ namespace HVACUnitarySystem {
 						TempMaxPLR = -0.1;
 						TempSysOutput = SensOutputOff;
 						TempLoad = ZoneLoad;
-						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 1.0 ) {
+						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 0.95 ) {
```

Comment:
The fact that the system had a different output capacity at the bottom end of speed = 2 (speed ratio = 0), this loop wound up at a PLR of 1.1 before exiting. All I did here was limit PLR to 1, and on the next line force PLR to explicitly = 1 (since I was seeing 0.9999 instead of 1 ... I wanted 1).